catalog.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>Поиск гостиницы в Седоне</title>
+
+
+  </head>
+<body>
+  <header class="page-header">
+    <nav class="main-navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item">
+          <a class="navigation-link" href="index.html" aria-current="page">Главная</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link navigation-link-current">Гостиницы</a>
```

Comment:
А если вы используете [ариа-куррент](https://www.aditus.io/aria/aria-current/), то тогда нужно ставить его активной ссылке
```suggestion
          <a class="navigation-link" href="index.html">Главная</a>
        </li>
        <li class="navigation-item">
          <a class="navigation-link" href="#">О Седоне</a>
        </li>
        <li class="navigation-item">
          <a class="navigation-link navigation-link-current" href="catalog.html"  aria-current="page">Гостиницы</a>
```